canonical-web-and-design/charmhub.io
# Implement fluid breakout layout on homepage

## Done
- _Implement fluid breakout layout on homepage._
- _Remove some unused CSS._
- _Update the maintenance template to share the same header strip as store._

## How to QA
- Go to https://charmhub-io-934.demos.haus and play with filters & search
- Everything should work as expected

## Issue / Card
Fixes #933 

## Screenshots
[if relevant, include a screenshot]
